types/node/querystring.d.ts

```diff
@@ -19,6 +19,14 @@ declare module "querystring" {
 
     function stringify(obj?: ParsedUrlQueryInput, sep?: string, eq?: string, options?: StringifyOptions): string;
     function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): ParsedUrlQuery;
+    /**
+     * The querystring.encode() function is an alias for querystring.stringify().
+     */
+    function encode(obj?: ParsedUrlQueryInput, sep?: string, eq?: string, options?: StringifyOptions): string;
```

Comment:
I think that the `obj` should not a optional,If ignore it,`encode` always returns an empty string.https://github.com/nodejs/node/blob/master/lib/querystring.js#L160-L197.